Automattic/newspack-plugin
# hotfix: check for woocommerce on password reset

### All Submissions:






### Changes proposed in this Pull Request:



Closes [0/1200550061930446/1206731853389179/f](https://app.asana.com/0/1200550061930446/1206731853389179/f).

This fixes a problem where sending a password reset email when WC is not installed results in a fatal:

![Screenshot 2024-03-06 at 16 42 06](https://github.com/Automattic/newspack-plugin/assets/17905991/c3d734af-ee6e-4589-965a-8a01fc1158d8)


### How to test the changes in this Pull Request:

1. On a test site, make sure Woo is not active
2. Go to any users Edit User page
3. Send a password reset link. On `release`, you will see a fatal error box appear (with no text). On this branch the generic lost password link will be sent

![Screenshot 2024-03-06 at 16 44 20](https://github.com/Automattic/newspack-plugin/assets/17905991/64e5fa04-8ed2-4312-abee-f1904def76dd)


### Other information:
